utils/docker/Dockerfile.focal

```diff
@@ -30,7 +30,7 @@ RUN mkdir /ms-playwright && \
     cd /ms-playwright-agent && \
     dotnet new console && \
     echo '<?xml version="1.0" encoding="utf-8"?><configuration><packageSources><add key="local" value="/tmp/"/></packageSources></configuration>' > nuget.config && \
-    dotnet add package Microsoft.Playwright && \
+    dotnet add package Microsoft.Playwright --prerelease && \
```

Comment:
the line above means: "hey use the /tmp/ directory as a package source". This line installs playwright, but since the local version is sometimes a pre-release, it did not use it, it went to nuget.org instead. That change fixes it so its picking up prereleases too.